### PR TITLE
Set of patches for TEC-480

### PR DESCRIPTION
**Pull request by umbobabo**
https://jira.economist.com/browse/TEC-480

### CODE REVIEW

File: user-menu.css [hunk 1]
```diff
@@ -58,6 +62,7 @@
   display: block;
   text-align: center;
   font-weight: bold;
+  background: var(--color-chicago);
```
Comment (keithamus) [inline user-menu.css]: Would be nice to have this overridable via var.

**Comment by keithamus** [comment]
Other than the above, LGTM.

File: index.css [hunk 2]
```diff
@@ -95,3 +96,17 @@
   overflow-y: auto;
   max-height: calc(100vh - var(--navigation__default-height));
 }
+
+@media (--mobile-up){
```
Comment (fabiosantoscode) [inline index.css]: I'd add a prefix to this variable. Not using prefixes has bitten us before.

**Comment by fabiosantoscode** [comment]
We are fully aware, but whatever is on top of the z-index stack is shitty third-party ad code.

**Comment by keithamus** [comment]
:disappointed:

**Comment by umbobabo** [comment]
Unfortunately highest z-index is unpredictable in the .com context.

**Comment by umbobabo** [comment]
Can we merge?

**Pull request merged**